t/lib/t/MusicBrainz/Server/Edit/Medium/SetTrackLengths.pm

```diff
@@ -104,9 +108,18 @@ test 'Fail gracefully if CD TOC has been removed' => sub {
 
     $c->sql->do("DELETE FROM cdtoc WHERE id = 1");
 
-    my $exception = exception { $edit->accept };
-    isa_ok $exception, 'MusicBrainz::Server::Edit::Exceptions::FailedDependency';
-    is $exception->message, localized_note('The CD TOC the track times were being set from has been removed since this edit was entered.');
```

Comment:
I had to change this test because with the added version field, `localized_note` could return the message & version keys in any order and this test would randomly fail.